parachains/runtimes/assets/westmint/src/weights/pallet_nft_fractionalization.rs

```diff
@@ -45,19 +45,69 @@
 use frame_support::{traits::Get, weights::Weight};
 use core::marker::PhantomData;
 
-/// Weight functions for `pallet_nfts`.
+/// Weight functions for `pallet_nft_fractionalization`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_nft_fractionalization::WeightInfo for WeightInfo<T> {
+	/// Storage: Nfts Item (r:1 w:0)
+	/// Proof: Nfts Item (max_values: None, max_size: Some(861), added: 3336, mode: MaxEncodedLen)
+	/// Storage: Balances Holds (r:1 w:1)
+	/// Proof: Balances Holds (max_values: None, max_size: Some(66), added: 2541, mode: MaxEncodedLen)
+	/// Storage: Nfts Collection (r:1 w:1)
+	/// Proof: Nfts Collection (max_values: None, max_size: Some(84), added: 2559, mode: MaxEncodedLen)
+	/// Storage: Nfts Attribute (r:1 w:1)
+	/// Proof: Nfts Attribute (max_values: None, max_size: Some(479), added: 2954, mode: MaxEncodedLen)
+	/// Storage: Assets Asset (r:1 w:1)
+	/// Proof: Assets Asset (max_values: None, max_size: Some(210), added: 2685, mode: MaxEncodedLen)
+	/// Storage: Assets Account (r:1 w:1)
+	/// Proof: Assets Account (max_values: None, max_size: Some(134), added: 2609, mode: MaxEncodedLen)
+	/// Storage: System Account (r:1 w:1)
+	/// Proof: System Account (max_values: None, max_size: Some(128), added: 2603, mode: MaxEncodedLen)
+	/// Storage: Assets Metadata (r:1 w:1)
+	/// Proof: Assets Metadata (max_values: None, max_size: Some(140), added: 2615, mode: MaxEncodedLen)
+	/// Storage: NftFractionalization NftToAsset (r:0 w:1)
+	/// Proof: NftFractionalization NftToAsset (max_values: None, max_size: Some(92), added: 2567, mode: MaxEncodedLen)
 	fn fractionalize() -> Weight {
-		// Minimum execution time: 44_312 nanoseconds.
-		Weight::from_parts(25_147_000, 3549)
-			.saturating_add(T::DbWeight::get().reads(2_u64))
-			.saturating_add(T::DbWeight::get().writes(5_u64))
+		// Proof Size summary in bytes:
+		//  Measured:  `462`
+		//  Estimated: `4326`
+		// Minimum execution time: 167_532_000 picoseconds.
+		Weight::from_parts(168_850_000, 0)
```

Comment:
weight increased by x7

Reply:
But this pallet was just merged in. Could there have been dummy weights before?

Reply:
yep, the previous weights were the dummy ones